app/models/risk_assessment.rb

```diff
@@ -4,7 +4,20 @@ def initialize(notice)
   end
 
   def high_risk?
-    risk_triggers.any? { |trigger| trigger.risky?(notice) }
+    risky = false
+
+    risk_triggers.each do |trigger|
+      trigger_risky = trigger.risky?(notice)
```

Comment:
Is it correct that this `each` logic boils down to `trigger.force_not_risky_assessment? && trigger.risky?(notice)`?

If so, I think `high_risk?` is just 
```
risk_triggers.any? do |trigger|
 trigger.force_not_risky_assessment? && trigger.risky?(notice)
end
```

Reply:
@thatandromeda no really, if any of `risk_triggers` is `force_not_risky_assessment?` and is `risky?` then the assessment is not `risky?`, it would be `risky?` in your example